level0_unit_tests_iris.log

```diff
@@ -75,35 +67,28 @@ The following tests FAILED:
-	588 - Unit_hipGetDeviceFlags_FuncTst (Failed)
-	596 - Unit_hipStreamPerThread_Basic (Failed)
-	597 - Unit_hipStreamPerThread_StreamQuery (Subprocess aborted)
-	598 - Unit_hipStreamPerThread_StreamSynchronize (Subprocess aborted)
-	599 - Unit_hipStreamPerThread_StreamGetPriority (Failed)
-	600 - Unit_hipStreamPerThread_StreamGetFlags (Failed)
-	602 - Unit_hipStreamPerThread_MemcpyAsync (Failed)
-	603 - Unit_hipStreamPerThread_EventRecord (Failed)
-	604 - Unit_hipStreamPerThread_EventSynchronize (Failed)
-	606 - Unit_hipStreamPerThread_DeviceReset_1 (SEGFAULT)
-	608 - Unit_hipGraphAddEmptyNode_Functional (Failed)
-	609 - Unit_hipGraphAddDependencies_Functional (Failed)
-	610 - Unit_hipGraph_BasicFunctional (Failed)
-	611 - Unit_hipGraph_SimpleGraphWithKernel (Failed)
-	612 - Unit_hipGraphAddMemcpyNode_Functional (Failed)
-	613 - ABM_AddKernel_MultiTypeMultiSize - int (Failed)
-	614 - ABM_AddKernel_MultiTypeMultiSize - long (Failed)
-	615 - ABM_AddKernel_MultiTypeMultiSize - float (Failed)
-	616 - ABM_AddKernel_MultiTypeMultiSize - long long (Failed)
-	617 - ABM_AddKernel_MultiTypeMultiSize - double (Failed)
-	620 - Stress_hipMemcpy_multiDevice-AllAPIs - int (Failed)
-	621 - Stress_hipMemcpy_multiDevice-AllAPIs - size_t (Failed)
-	622 - Stress_hipMemcpy_multiDevice-AllAPIs - long double (Subprocess killed)
-	643 - stream (SEGFAULT)
-	660 - FloydWarshall (Timeout)
-	666 - PrintfDynamic (Failed)
-	677 - sycl_chip_interop (SEGFAULT)
-	678 - sycl_chip_interop_usm (SEGFAULT)
\ No newline at end of file
+	560 - Unit_hipOccupancyMaxPotentialBlockSize_rangeValidation (Failed)
+	575 - Unit_hipDeviceGetCacheConfig_FuncTst (Failed)
+	586 - Unit_hipGetDeviceFlags_FuncTst (Failed)
+	594 - Unit_hipStreamPerThread_Basic (Failed)
+	595 - Unit_hipStreamPerThread_StreamQuery (Subprocess aborted)
+	596 - Unit_hipStreamPerThread_StreamSynchronize (Subprocess aborted)
+	597 - Unit_hipStreamPerThread_StreamGetPriority (Failed)
+	598 - Unit_hipStreamPerThread_StreamGetFlags (Failed)
+	600 - Unit_hipStreamPerThread_MemcpyAsync (Failed)
+	601 - Unit_hipStreamPerThread_EventRecord (Failed)
+	602 - Unit_hipStreamPerThread_EventSynchronize (Failed)
+	606 - Unit_hipGraphAddEmptyNode_Functional (Failed)
+	607 - Unit_hipGraphAddDependencies_Functional (Failed)
+	608 - Unit_hipGraph_BasicFunctional (Failed)
+	609 - Unit_hipGraph_SimpleGraphWithKernel (Failed)
+	610 - Unit_hipGraphAddMemcpyNode_Functional (Failed)
+	611 - ABM_AddKernel_MultiTypeMultiSize - int (Failed)
+	612 - ABM_AddKernel_MultiTypeMultiSize - long (Failed)
+	613 - ABM_AddKernel_MultiTypeMultiSize - float (Failed)
+	614 - ABM_AddKernel_MultiTypeMultiSize - long long (Failed)
+	615 - ABM_AddKernel_MultiTypeMultiSize - double (Failed)
+	618 - Stress_hipMemcpy_multiDevice-AllAPIs - int (Failed)
+	619 - Stress_hipMemcpy_multiDevice-AllAPIs - size_t (Failed)
+	620 - Stress_hipMemcpy_multiDevice-AllAPIs - long double (Failed)
+	648 - hipMultiThreadAddCallback (Not Run)
```

Comment:
not run? @franz

Reply:
Is that the logfile in the repository, or is that from an actual `ctest` run ? I have checked (by running the tests) and it's enabled. I don't generally update the logfile in the repo as i don't have either iris or arcticus hardware.